components/tooltip/index.jsx

```diff
@@ -161,6 +178,9 @@ class PopoverTooltip extends React.Component {
 			this.props.isOpen === undefined ? this.state.isOpen : this.props.isOpen;
 		const align = this.props.align;
 
+		// REMOVE AT NEXT BREAKING CHANGE
```

Comment:
When is "the next breaking change"? Maybe we should be more proscriptive about when this will get removed?

Reply:
We don't have a date at this point. We could say remove at 1.0 or 0.9?

Reply:
Added version to comment